lib/data/models/connection_data.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'connection_data.freezed.dart';
+
+@freezed
+class ConnectionData with _$ConnectionData {
+  const factory ConnectionData.gql({
+    required String name,
+    required int networkId,
+    required String group,
+    required List<String> endpoints,
+    required int timeout,
```

Comment:
I thins any time-related things should be unified. For example, in milliseconds with 'ms' suffix. But much better to use Duration because it provides all math for time calculations.

Reply:
-